DataFormats/Detectors/ITSMFT/common/include/DataFormatsITSMFT/TrkClusRef.h

```diff
@@ -26,11 +26,34 @@ namespace itsmft
 struct TrkClusRef : public o2::dataformats::RangeRefComp<4> {
   using o2::dataformats::RangeRefComp<4>::RangeRefComp;
   uint16_t pattern = 0; ///< layers pattern
+  uint32_t clsizes = 0; ///< cluster sizes for each layer
```

Comment:
Shouldn't we swap the order of `pattern` and `clsizes` to avoid wasting 16 bits in padding per ref?

Reply:
done, thanks for spotting!